org.alloytools.alloy.application/src/main/java/ca/uwaterloo/watform/dash4whole/RapidDash.java

```diff
@@ -63,17 +68,51 @@ public static void main(String args[]) throws Exception {
             System.out.println("=========== Parsing+Typechecking " + fileName + " =============");
 
             // Solve the Dash module first to get initial values
-            DashModule dash = DashUtil.parseEverything_fromFileDash(rep, null, actual);
+            DashModule dash;
+            String addCommand = "";
+            try {
+                try {
+                    System.out.println("You can specify the scopes in a Dash command, for example, \"run {} for exactly 4 Medication, exactly 4 Patient\".");
+                    System.out.println("Please specify the command you want to run with the Dash model(leave it blank to run the default command):");
+                    addCommand = sc.nextLine().trim();
+                    if (!addCommand.endsWith("\n")) {
+                        addCommand += "\n";
+                    }
+                    Files.write(path, addCommand.getBytes(), StandardOpenOption.APPEND);
+                } catch (IOException e) {
+                    System.err.println("File does not exist.");
+                    exit(1);
+                }
+                dash = DashUtil.parseEverything_fromFileDash(rep, null, actual);
+            } finally {
+                // Remove the added command
+                RandomAccessFile f = new RandomAccessFile(actual, "rw");
+                long length = f.length() - 1;
+                byte b;
+                do {
+                    length -= 1;
+                    f.seek(length);
+                    b = f.readByte();
+                } while (b != 10);
+                f.setLength(length + 1);
+                f.close();
+            }
```

Comment:
Why ask for a command in the cmd line interface instead of asking the user to write down the command in the dash file?
I parsing command directly from the dash file is better because of two reasons:
1. the user can write the command to the file, and they don't need to re-type the command everytime they run RapidDash
2. We can avoid this awkward writing to dash file then removing from dash file logic.

Reply:
That makes sense! I've made the requested change.